### DISCUSSION TITLE
configurazione SSL

### DISCUSSION THREAD

**Discussion by Virgilio123** [indomus/forum#1210] [GENERALE - Discussioni generiche]
Buongiorno a Tutti e grazie per l'aiuto che vorrete darmi.

ho installa to Home Assistant usando docker su un sistema armbian, aggiornando all'ultima versione.

ho configurato sette sonoff mini usando il tasmoda e tutto funziona.
la mia istanza di home assistant veniva da me raggiunta all'indirizzo ip del 192.168.1.xxx:8123

volevo attivare la connessione sicura, perchè vorrei configurare l'integrazione ALEXA.

ho seguito le istruzioni indicate in questo sito -

**ma ora ho due problemi:**

1) riesco a collegarmi alla mia istanza di home assistan, ma solo utilizzando il link esterno con protocollo ssl, mentre il collegamento usando la rete locale non funziona più!!!

2) non sono riuscito a far funzionare lo script indicato nell'articolo. in particolare il comando `systemctl restart home-assistant@homeassistant.service` non funziona perchè io non ho un servizio homeassistant e nel docker il comando systemctl non è disponibile.

Virgilio

**Comment by Scialla** [comment]
però, se segui la guida di un altro sito, chiedi aiuto a loro.... altrimenti riparti seguendo una delle guide di inDomus
inoltre è normale che, una volta configurato l'accesso con dehydrated, l'accesso in locale dia errore, il certificato rilasciato e sull'FQDN e quindi l'ip interno (cui devi accedere necessariamente in https) darà errore, dovrai aggiungerlo alle eccezioni e accederai ugualmente.
Se non vuoi errori devi usare un reverse proxy, tipo caddy

**Reply by Virgilio123** [reply]
Grazie per la risposta.
in effetti dopo un pò di tentativi sono riuscito a configurare un server apache che fa da reverse proxy.
